wagtail_transfer/views.py

```diff
@@ -161,4 +188,21 @@ def do_import(request):
 
     importer.run()
 
-    return redirect('wagtailadmin_explore', request.POST['dest_page_id'])
+    if dest_page_id:
+        return redirect('wagtailadmin_explore', dest_page_id)
+    else:
+        return redirect('wagtailadmin_explore', Page.objects.first().id)
+
+
+def check_page_existence_for_uid(request):
+    """
+    Check whether a page with the specified UID exists - used for checking whether a page has already been imported
+    to the destination site
+    """
+    uid = request.GET.get('uid', '')
+    locator = get_locator_for_model(Page)
+    page_exists = True if locator.find(uid) else False
+
+    return JsonResponse({
```

Comment:
Might be a little more RESTy to make this return a 404 status code if the uid doesn't exist and 200 if it does.